Tests/TicTacToeTest.java

```diff
@@ -0,0 +1,76 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
```

Comment:
Consider adding more test cases to cover different scenarios, such as checking for a draw, testing the game reset functionality, and verifying the behavior when a player tries to make a move in an already occupied cell.